src/Admin/Content/StaticContentAdmin.php

```diff
@@ -11,22 +11,42 @@
 
 namespace Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Admin\Content;
 
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
 use Sonata\AdminBundle\Datagrid\DatagridMapper;
 use Sonata\AdminBundle\Datagrid\ListMapper;
 use Sonata\AdminBundle\Form\FormMapper;
 use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContentBase;
 use Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Admin\AbstractAdmin;
 use Symfony\Cmf\Bundle\TreeBrowserBundle\Form\Type\TreeSelectType;
-use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
 
 class StaticContentAdmin extends AbstractAdmin
 {
     protected $translationDomain = 'CmfSonataPhpcrAdminIntegrationBundle';
 
+    /**
+     * Configuration, that can be passed to CKEditorType.
+     *
+     * @var []
+     */
+    private $ckEditorConfig;
+
     public function getExportFormats()
     {
-        return array();
+        return [];
+    }
+
+    /**
+     * Configuration passed to the CKEditorType. Have a look at the docs:
```

Comment:
styleci complains. lets move "Have a look at the docs:" to a second section and do something like "The configuration array is explained in the IvoryCKEditorBundle documentation: http://"